_data/changelogs/component-checkbox.yml

```
@@ -2,6 +2,14 @@ title: Checkbox
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added indeterminate styles for checkboxes.
+    summaryAdditional: Checkboxes will now display as indeterminate when `input.indeterminate = true` is set via JS or the `data-indeterminate` attribute is present. This is currently only a style addition and does not affect checkbox functionality.
```

Recommend we expand the JS abbreviation and move away from passive voice. 
```suggestion
    summaryAdditional: Checkboxes will now display as indeterminate when you set `input.indeterminate = true` via JavaScript or add the `data-indeterminate` attribute. This is currently only a style addition and does not affect checkbox functionality.
```

---

Sounds good! Thanks for the tip